tests/unit/EntityInfo.spec.ts

```diff
@@ -248,12 +249,19 @@ describe('EntityInfo - Click Tests - Alterations', () => {
     sessionStorage.clear()
     sessionStorage.setItem('EDIT_URL', `${process.env.VUE_APP_PATH}/edit/`)
     sessionStorage.setItem('BUSINESS_ID', 'BC1234567')
-    sessionStorage.setItem('SHOW_COMPANY_INFO_BUTTON', 'yes') // FUTURE: remove this
     store.state.entityIncNo = 'BC1234567'
     // store.state.entityType = 'LTD' // FUTURE: uncomment this
 
     const router = mockRouter.mock()
-    const wrapper = mount(EntityInfo, { vuetify, store, router })
+    const wrapper = mount(EntityInfo, {
+      vuetify,
+      store,
+      router,
+      computed: {
+        // mock this getter to override FF check
+        viewChangeInfoEnabled () { return true }
```

Comment:
I could not figure out (quickly) how to mock the global FF getter, so I mocked this component getter instead.